iree/hal/api.cc

```diff
@@ -606,9 +1097,19 @@ IREE_API_EXPORT iree_host_size_t IREE_API_CALL iree_hal_buffer_view_shape_dim(
   return buffer_view->shape[index];
 }
 
+IREE_API_EXPORT iree_host_size_t
+iree_hal_buffer_view_element_count(const iree_hal_buffer_view_t* buffer_view) {
+  CHECK(buffer_view) << "NULL buffer_view handle";
```

Comment:
return `IREE_STATUS_INVALID_ARGUMENT` instead?

Reply:
this is an accessor property and doesn't return a status (it's equivalent to a pointer dereference)